subsys/bluetooth/controller/include/ll_feat.h

```diff
@@ -276,3 +273,17 @@
 				  LL_FEAT_BIT_ISO_BROADCASTER | \
 				  LL_FEAT_BIT_SYNC_RECEIVER | \
 				  LL_FEAT_BIT_PERIODIC_ADI_SUPPORT)
+
+/* Connected Isochronous Stream (Host Support) bit is controlled by host */
+#if defined(CONFIG_BT_CTLR_CONN_ISO)
+#define LL_FEAT_HOST_BITS_ISO_CHANNELS BIT64(BT_LE_FEAT_BIT_ISO_CHANNELS)
```

Comment:
The naming of `ISO_CHANNELS` in this context is actually pretty confusing, as it is `CIS Host Support`, and doesn't relate to broadcast isochronous channels. We should rename it, but that can done in a separate PR